### PR TITLE
Add doc for optional parameters on extraMounts

### PR DESCRIPTION
**Pull request by sestegra**
After diving into the source code, I updated the extraMounts example to show all parameters.

### CODE REVIEW

**Comment by aojea** [comment]
I think we should match the API docs
https://pkg.go.dev/sigs.k8s.io/kind@v0.11.1/pkg/apis/config/v1alpha4#Mount

**Comment by sestegra** [comment]
As a user of this documentation, I would see this documentation as an end-user reference guide.
End-user probably doesn't understand a GO struct.

**Comment by aojea** [comment]
> As a user of this documentation, I would see this documentation as an end-user reference guide.
> End-user probably doesn't understand a GO struct.

I mean the text on the comments :smile:

**Comment by aojea** [comment]
> > As a user of this documentation, I would see this documentation as an end-user reference guide.
> > End-user probably doesn't understand a GO struct.
> 
> I mean the text on the comments

oh, they already match , sorry
/ok-to-test
/lgtm

File: site/static/examples/config-with-mounts.yaml [hunk 1]
```diff
@@ -5,4 +5,14 @@ nodes:
   # add a mount from /path/to/my/files on the host to /files on the node
   extraMounts:
   - hostPath: /path/to/my/files/
-    containerPath: /files
\ No newline at end of file
+    containerPath: /files
+    # optional: if set, the mount is read-only.
+    # false is the current default
```
Comment (aojea) [inline site/static/examples/config-with-mounts.yaml:10]: does this reads better this way?
```suggestion
    # default false
```

Comment (sestegra) [reply site/static/examples/config-with-mounts.yaml]: Update done in last commit

**Comment by aojea** [comment]
/lgtm
/approve
Thanks

**Pull request merged**